cppwinrt/cmd_reader.h

```diff
@@ -153,13 +145,13 @@ namespace cppwinrt
         HKEY key;
 
         if (0 != RegOpenKeyExW(
-            HKEY_LOCAL_MACHINE,
-            L"SOFTWARE\\Microsoft\\Windows Kits\\Installed Roots",
-            0,
-            // https://task.ms/29349404 - The SDK sometimes stores the 64 bit location into KitsRoot10 which is wrong,
-            // this breaks 64-bit cppwinrt.exe, so work around this by forcing to use the WoW64 hive.
-            KEY_READ | KEY_WOW64_32KEY, 
-            &key))
+                     HKEY_LOCAL_MACHINE,
```

Comment:
Not against it. But I found this big indent curious. My only idea is that it's applying a 4-space indent from the beginning of `RegOpenKeyExW`, instead of `0`.